pages/views/sequences.py

```diff
@@ -80,7 +86,14 @@ def get_object(self, queryset=None):
         sequence_slug = self.kwargs.get('sequence')
         rank = self.kwargs.get('rank')
         sequence = get_object_or_404(Sequence, slug=sequence_slug)
-        return get_object_or_404(EnumeratedElements, sequence=sequence, rank=rank)
+        element = get_object_or_404(EnumeratedElements, sequence=sequence, rank=rank)
+
+        element.is_live_event = LiveEvent.objects.filter(element=element.page_element).exists()
```

Comment:
Somehow it feels this code and the similar code above should be in a `decorate_element` method defined in a common mixin.